indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -596,7 +599,8 @@ def _default_template_id(self):
         return str(self.regform.ticket_template_id) if self.regform.ticket_template_id else None
 
     def _filter_registrations(self, registrations):
-        return [r for r in registrations if not r.is_ticket_blocked]
+        template_is_ticket = DesignerTemplate.get_or_404(self.template_id).is_ticket
+        return [r for r in registrations if not template_is_ticket or not r.is_ticket_blocked]
```

Comment:
This doesn't work. You do not know which ticket template will be used, because there's a dropdown in the dialog that lets you choose it. And currently the code fails because `self.template_id` can be None (easiest way to reproduce: create a new event, create a regform, register, and go to "print tickets" in the list of registrants, without ever touching ticket settings)

(not urgent, can be fixed in the new year, just saw an error on [sentry](https://sentry.getindico.io/indico-team/indico/issues/8981/) from someone's test event)

Reply:
OK, this is actually quite easy to fix, because when we are in this RH, we only deal with ticket templates (the dropdown to choose the template does not offer non-ticket templates). So we can just revert the change here :)

Reply:
fixed in 84901d6a2b